source/mainnet/net/resources/ccd-scan-staking.rst

```diff
@@ -0,0 +1,118 @@
+.. _bakers-view:
+
+========================
+CCDScan Staking overview
+========================
+
+The Staking overview shows information about bakers during the selected time range in the filter:
+
+.. image:: ../images/ccd-scan/ccd-scan-bakers.png
+
+|
+
+The graphs show when the next payday happens, the amount of staked CCD, the number of bakers, and the total of rewards during the selected time range in the filter. The **Previous payday block** contains information about all rewards paid out for a payday period. If you click **Previous payday block**, a special block details screen for the previous pay day block appears.
+
+.. image:: ../images/ccd-scan/ccd-scan-staking-payday-block-details.png
+
+|
+
+The Pay Day block details contains the following:
+
+- **Block**: the first six digits of the block hash. Click Copy |copy| to copy the entire hash. The block status is also shown.
+- **Age**: the age of the block and date/time stamp when it was created.
+- **Baker id**: ID of the baker who baked the block. Click the baker IDto see the :ref:`baker details<home-screen-baker>`.
+- **Tokenomics**: contains the following details about the block:
+    - Distributed minted CCD: Contains the total minted CCDs and a breakdown on their distribution across the baker reward account, finalization reward account, and foundation account.
+    - Accrued block rewards: similar to a ‘regular’ :ref:`block<home-screen-block>`; contains the accrued block rewards.
+    - Payday foundation rewards: displays the payout to the foundation account for the pay day period and contains the option to drill-through to the :ref:`foundation account<accounts-view>`.
```

Comment:
```suggestion
    - Payday foundation rewards: displays the payout to the foundation account for the payday period and contains the option to drill-through to the :ref:`foundation account<accounts-view>`.
```
Should we use `payday` everywhere?

Reply:
We have used pay day everywhere else in the documentation. However, on the CCDScan screen it is written as payday so I have to write it as readers see it.

Reply:
https://github.com/Concordium/concordium-scan/issues/2